msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md

```diff
@@ -1,63 +1,85 @@
 
 ## Improved Windows broker experience
 
-Recently, an updated authentication broker experience was made available for Windows systems. The new broker is written in C++, is well tested, and is significantly more performant and secure. One of the biggest consumers of the new broker experience is the Microsoft 365 suite of apps.
+Latest Windows releases include an updated WAM. The new broker is written in C++, is well-tested, and is significantly more performant and secure. One of the biggest consumers of the new broker experience is the Microsoft 365 suite of apps.
 
 > [!IMPORTANT]
-> With the introduction of the updated broker, we have also updated MSAL.NET to expose its capabilities under a simplified API. Starting with version 4.52.0, the improved broker experience in MSAL.NET will replace the existing broker flows by default.
+> With the introduction of the updated broker, we have also updated MSAL.NET to expose its capabilities under a simplified API. Starting with version 4.52.0, the new broker will be the default in MSAL.NET.
 
-The improved WAM broker fixes a number of issues with the legacy WAM implementation and provides other benefits:
+The new WAM fixes a number of issues with the legacy implementation and provides other benefits, including:
 
 - New implementation is more stable, easier to add new features, and has less chance of regressions.
-- Works in apps that are run-as-admin.
-- Adds support for Proof-of-Possession tokens.
-- Fixes assembly size issues.
+- Works in apps that are ran under the Administrator user context.
+- Adds support for [Proof-of-Possession tokens](../../advanced/proof-of-possession-tokens.md).
+- Decreases assembly size.
 
 ## Enabling WAM
 
-Due to platform-specific and backwards compatibility requirements, WAM implementation resides in three packages:
+Due to platform-specific and backwards compatibility requirements, WAM support is split across three packages:
 
-- [Microsoft.Identity.Client](https://www.nuget.org/packages/Microsoft.Identity.Client/)(i.e. MSAL)
-- [Microsoft.Identity.Client.Broker](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/) - WAM support
-- [Microsoft.Identity.Client.Desktop](https://www.nuget.org/packages/Microsoft.Identity.Client.Desktop/) - WAM and WebView2 support
+- [Microsoft.Identity.Client](https://www.nuget.org/packages/Microsoft.Identity.Client/) (i.e., MSAL) - core library for token acquisition.
+- [Microsoft.Identity.Client.Broker](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/) - adds support for authentication with the broker.
+- [Microsoft.Identity.Client.Desktop](https://www.nuget.org/packages/Microsoft.Identity.Client.Desktop/) - adds support for authentication with the broker on legacy platforms, like .NET Framework, .NET Core 3.1, and UWP.
```

Comment:
we should be able to use new broker in .NET FWK and .Net Core 3.1

Reply:
@gladjohn - would that be through `Microsoft.Identity.Client.Desktop` or are you saying that we can natively do it through the broker package? If that is the case, what is the purpose of the desktop package?